### ISSUE TITLE
Use in windows enviroment

### ISSUE THREAD

**Issue by george-karma** [StanislasBertrand/RetinaFace-tf2#15]
Hi, I am trying to use this in a windows enviroment. Running the Makefile with GnuWin32  only show the commands in the makefile. Running the commands manually in the rcnn folders specified returns:

AttributeError: 'MSVCCompiler' object has no attribute 'compiler_so'

**Comment by StanislasBertrand** [comment]
Hi,

Sorry but I have absolutely no knowledge about developing tensorflow applications in windows :( 
If you do end up finding how to make it work I'd greatly appreciate a PR though !

**Comment by george-karma** [comment]
Im gonna keep this issue open for now and close it once i figure it out or give up in a few days.

**Comment by george-karma** [comment]
I managed to build the cython related codes and sidestep the compiler issue using [this method](https://github.com/deepinsight/insightface/issues/679#issuecomment-495870070)

For me this results in a `CUBLAS_STATUS_NOT_INITIALIZED` error . This error can be solved in one of two ways depending on what works for you.

1.  Enable memory growth by adding the following code in `retinaface.py` bellow the tf import:
 ```python
gpus = tf.config.experimental.list_physical_devices('GPU')
if gpus:
    try:
        for gpu in gpus:
            tf.config.experimental.set_memory_growth(gpu, True)
    except RuntimeError as e:
        print(e)
```
This will make it so the tensorflow process attempts to only allocate as much memory as needed. For me this results in `CUDA_ERROR_OUT_OF_MEMORY`. Using the nvidia-smi shows that tensorflow attempts to allocate all the memory you have, which runs into issues if you have other applications using your gpu at the same time. Im not sure if this is a tensorflow issue or just my setup.

2. Limit the amount of memory available to tensorflow by adding this code instead:
```python
gpus = tf.config.experimental.list_physical_devices('GPU')
if gpus:
    try:
        for gpu in gpus:
            tf.config.experimental.set_virtual_device_configuration( gpus[0],[tf.config.experimental.VirtualDeviceConfiguration(memory_limit=1024)])
    except RuntimeError as e:
        print(e)
```
I have a 8gb RTX 2070 and limit the available memory to 6.5gb. I haven't had any issues after a day of tesing. (cuda 11.2, cudnn 8.0.5, python 3.7)
